mice-and-cats-java/src/main/java/foop/views/EntreView.java

```diff
@@ -0,0 +1,101 @@
+package foop.views;
+
+import foop.message.CreateGameMessage;
+import foop.message.InitialMessage;
+import lombok.extern.slf4j.Slf4j;
+
+import javax.swing.*;
+import javax.swing.event.DocumentEvent;
+import javax.swing.event.DocumentListener;
+import java.awt.*;
+
+@Slf4j
+public class EntreView extends JPanel implements View {
+
```

Comment:
Did you mean EntryView?

I would have named this one StartView or TitleScreenView and the other one JoinGameView